src/othermbc.md

```diff
@@ -40,70 +41,75 @@ nonsensical.
 
 The mapper does not support an outer bank for battery SRAM.
 
-To start a game, do the following in code run from RAM: Write $A5 to
-$1000, write game starting bank number to $2000, write any value to
-$7000, write $98 to $1000, write $01 to $2000 (so that 32K games
-work), jump to $0100.
+To start a game, do the following with code run from RAM:
+1. Write $A5 to $1000
+2. Write game's first bank number to $2000
+3. Write any value to $7000
+4. Write $98 to $1000
+5. Write $01 to $2000 (so that 32K games work)
+6. Jump to $0100
 
 ## Wisdom Tree
 
-The Wisdom Tree mapper is a simple, cost-optimized one chip design
-consisting of a 74LS377 octal latch, aside from the ROM chip. Because
+The Wisdom Tree mapper is a simple, cost-optimized one-chip design
+consisting of a 74LS377 octal latch in addition to the ROM chip. Because
 the mapper consists of a single standard 74 series logic chip, it has
 two unusual properties:
 
-First, unlike a usual MBC, it switches the whole 32 kiB ROM area instead
-of just the $4000-$7FFF area. If you want to use the interrupt vectors
+First, unlike a usual MBC, it switches the whole 32 KiB ROM area instead
+of just the $4000-$7FFF area. Therefore, if you want to use [the interrupt vectors](<#Interrupt Handling>)
 with this cart, you should duplicate them across all banks.
 Additionally, since the initial state of the '377 can't be guaranteed,
 the ROM header and some code for switching to a known bank should also
 be included in every bank. This also means that the Wisdom Tree mapper
-could be used as a multicart mapper for 32 kiB ROMs, assuming there was
+could be used as a multicart mapper for 32 KiB ROMs, assuming there was
 enough ROM space in each bank for some small initialization code, and
 none of the ROMs wrote to the $0000-$7FFF area. For example, if the
 last 5 bytes of all banks are unused, games can be patched as follows:
 
-    ; At $0100 in all banks but the first
-      nop
-      jp $7FFB
-
-    ; At $7FFB in all banks
-      ld hl, $0100
-      ld [hl], a
-      jp hl
-
-Second, because the '377 latches data on the *positive* edge, and the
-value on the Game Boy data bus is no longer valid when the positive edge
-of the write pulse arrives, the designer of this mapper chose to use the
+```rgbasm
+; At $0100 in all banks but the first
+    nop
+    jp $7FFB
+```
+```rgbasm
+; At $7FFB in all banks
+    ld hl, $0100
+    ld [hl], a
+    jp hl
+```
+
+Second, because the '377 latches data on the *positive* write pulse edge,
```

Comment:
I think a small reference to how positive-edge triggered latches should be linked (e.g. https://www.allaboutcircuits.com/textbook/digital/chpt-10/edge-triggered-latches-flip-flops/)